### DISCUSSION TITLE
2023.X Releases?

### DISCUSSION THREAD

**Discussion by amodelandme** [QuestPDF/QuestPDF#544] [General]
Are there any plans for more releases after 2022.12?

**Comment by MarcinZiabek** [comment]
Short answer: Yes 😁  

I am making the final touches on the 2023.4 release. It does not contain any features but introduces the split licensing model following all details available on the webpage. This change is about ensuring funding for the project and its stability. Would you like to take a look at this comment? https://github.com/QuestPDF/QuestPDF/discussions/491#discussioncomment-5642899 I am looking for feedback regarding the idea of throwing a license-oriented exception to better communicate the license change.

Not long after, I plan to introduce a next 2022.5 release with various optimizations for images and support for merging documents. 

Changing the license was challenging, but now I expect that the realese peace will come back to monthly or bi-monthly.

In case of any known bugs, I will do my best to continue support of the `2022.12` version and provide quality updates for a couple of months more.

**Reply by amodelandme** [reply]
Thanks for the response.

**Comment by douglott** [comment]
I've already paid my license fee!  Amazing software...looking forward to the upcoming releases.

**Reply by MarcinZiabek** [reply]
I'm glad to hear that 😁 So far, you are the first and only person to purchase the license. Thank you! Would you please share your comments regarding the license-oriented exception? I am still unsure if I should follow that path 🤣https://github.com/QuestPDF/QuestPDF/discussions/491#discussioncomment-5642899

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
